routers/api/v1/api.go

```diff
@@ -455,6 +455,9 @@ func RegisterRoutes(m *macaron.Macaron) {
 				m.Combo("/:username").Get(org.IsMember).
 					Delete(reqToken(), reqOrgOwnership(), org.DeleteMember)
 			})
+			m.Group("/repos", func() {
```

Comment:
Just as a side note during review context we should use a combo and migrate https://github.com/go-gitea/gitea/blob/master/routers/api/v1/api.go#L335 that does not currently follow the normal path. Still, this will maybe break few things so I will wait for the advice of other before doing that. Maybe in an other PR.